### ISSUE TITLE
Limited experience with Lambda Triggers and Destinations in AWS Amplify

### ISSUE THREAD

**Issue by vbbandeira** [aws-amplify/amplify-js#8667]
### Is this related to a new or existing API?

Interactions

### Is this related to another service?

AWS Lambda Triggers

### Describe the feature you'd like to request

Hello everyone, I'm trying to set up a project within AWS Amplify to automate the creation of an Events framework based on Lambda functions, with the intention of deploying a backup of the event framework.

In the project, my goal is to add a trigger from the Alexa Skill Kit, to trigger a lambda function that targets the AmazonEvent Bridge.

However, when I try to configure within AWS Amplify my settings for triggers are limited to the web application itself and in user-dependent situations like login authentication, email verification, and the like.

Does anyone know if it's possible to add extrinsic triggers to the application and if it's possible to add destinations for when a lambda function is executed because I couldn't identify this configuration option in any step within AWS Amplify? If so, which path should I follow to achieve this result?

This image below is how it was done in the original event-based project, I want to replicate something like this within Amplify.

<img width="854" alt="Screen Shot 2021-07-29 at 16 19 02" src="https://user-images.githubusercontent.com/25926739/127553883-ff1bf23c-ae27-4c25-b778-7dfaf9bee2a8.png">

Thanks!

### Describe the solution you'd like

I would like to have a more comprehensive list of trigger options for lambdas, as the Lambda service within AWS makes available. Because the services I'm trying to use as a trigger for a lambda function are apparently not available.

### Describe alternatives you've considered

I can only configure the trigger and destination for lambda within the AWS Lambda console itself, I thought I would be able to do this within AWS Amplify since the service is provided by the same company.

### Is this something that you'd be interested in working on?

- [X] 👋 I may be able to implement this feature request
- [ ] ⚠️ This feature might incur a breaking change

**Comment by hoan-pham-duy** [comment]
Hi, have you tried to go to the Amplify's function, provide needed permissions, and add triggers in the web browser?
After you can config it, you can add trigger and destination in Lambda cloud formation template in amplify

**Comment by vbbandeira** [comment]
> Hi, have you tried to go to the Amplify's function, provide needed permissions, and add triggers in the web browser?
> After you can config it, you can add trigger and destination in Lambda cloud formation template in amplify

Yes, I can do this in the web browser, but this is not so beneficial if I need to recreate the project multiple times using Amplify deploy. Because that way I would have to go to the web browser every time to add the triggers and that would take me a long time, you know?

**Comment by hoan-pham-duy** [comment]
> ause that way I would have to go to the web browser every time to add the triggers and that would take me a long time, you know?

yes, if you can config it manually, you should modify Cloud Formation Template of Lambda Functions inside Amplify Project, 
Here is an example related to modify Cloud Formation Template:
https://medium.com/@navvabian/how-to-add-an-sqs-queue-to-your-amplify-cli-bootstrapped-project-cb7781c636ed
